### PR TITLE
cookbook: Improve the introduction page

### PR DESCRIPTION
**Pull request by tcharding**
Written language is hard, continued improvement is required and likely many iterations.

Attempt to improve the introduction page.

### CODE REVIEW

**Comment by tcharding** [comment]
@realeinherjar are you able/willing to review this one? If so we can test out the github action if/when it merges to see if we have things configured correctly now.

File: cookbook/src/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Getting Started
+
+To add `rust-bitcoin` to your project, run:
+
+```bash
+cargo add bitcoin
+```
+
+If you are playing around you probably want to use the "rand-std" feature so you can generate random
+keys while you do so.
```
Comment (realeinherjar) [inline cookbook/src/getting_started.md:10]:
```suggestion
keys:
```

Less is more

File: cookbook/src/intro.md [hunk 2]
```diff
@@ -1,36 +1,25 @@
-# Rust Bitcoin
+# Welcome!
 
-[![CC0 1.0][cc-shield]][cc]
+This book is created and maintained by those involved in the
+[`rust-bitcoin`](https://github.com/rust-bitcoin) GitHub organization, contributions are
+appreciated. It covers various crates from the org and as such, aims to be useful to developers
+wanting to write code in Rust that interacts with the Bitcoin network. It is specifically not
+limited to just the [`rust-bitcoin`](https://github.com/rust-bitcoin/rust-bitcoin) crate, although
+that is a good starting point if you want a one stop shop for interacting with Bitcoin in Rust.
```
Comment (realeinherjar) [inline cookbook/src/intro.md:8]:
```suggestion
that is a good starting point if you want a one-stop-shop for interacting with Bitcoin in Rust.
```

File: cookbook/src/intro.md [hunk 3]
```diff
@@ -1,36 +1,25 @@
...
 
-[`rust-bitcoin`](https://github.com/rust-bitcoin/rust-bitcoin) is a library for working with Bitcoin in Rust.
-It contains Bitcoin network protocol and associated primitives.
-You can find more by reading the [documentation](https://docs.rs/bitcoin).
+There are a number of good libraries built outside of the `rust-bitcoin` organization that build on
+top of the crates covered here, two that you might like to check out are:
```
Comment (realeinherjar) [inline cookbook/src/intro.md:11]:
```suggestion
There are a number of good libraries outside of `rust-bitcoin` organization that uses the crates covered here, two that you might like to check out are:
```

File: cookbook/src/intro.md [hunk 4]
```diff
@@ -1,36 +1,25 @@
...
-```
-
-This cookbook provides straightforward examples that showcase effective approaches
-for accomplishing typical Bitcoin-related programming tasks,
-and utilizing the Rust ecosystem's crates.
-
-The book covers various topics, including receiving data over P2P,
-parsing blocks and transactions,
-and constructing and signing transactions.
+Finally, this book is currently WIP but hopes to eventually cover various topics, including parsing
```
Comment (realeinherjar) [inline cookbook/src/intro.md:16]:
```suggestion
Finally, this book is currently Work in Progress (WIP) but hopes to eventually cover various topics, including parsing
```

Don't assume people know all TLAs

File: cookbook/src/intro.md [hunk 5]
```diff
@@ -1,36 +1,25 @@
...
+blocks and transactions, constructing and signing transactions, receiving data over P2P, plus fun
```
Comment (realeinherjar) [inline cookbook/src/intro.md:17]:
```suggestion
blocks and transactions, constructing and signing transactions, receiving data over Peer-to-Peer (P2P), plus fun
```

TLAs

**Review by realeinherjar — APPROVED** [review]
Some suggestions that don't block merging...

**Comment by tcharding** [comment]
All good points, will work them in and re-spin. Thanks for the review!

**Comment by tcharding** [comment]
I'm a goose, this PR needs migrating over to https://github.com/rust-bitcoin/rust-bitcoin.github.io because we are going to archive this repo.

File: cookbook/src/intro.md (hunk 4 above)
Comment (tcharding) [reply cookbook/src/intro.md:16]: Used:
```
Finally, this book is currently a work inn progress but hopes to eventually cover various topics,
including parsing blocks and transactions, constructing and signing transactions, receiving data
over the peer-to-peer network, plus fun stuff you can do with miniscript.
```

File: cookbook/src/intro.md (hunk 3 above)
Comment (tcharding) [reply cookbook/src/intro.md:11]: Used:
```
There are a number of good libraries outside of the `rust-bitcoin` organization that use the crates
covered here, two that you might like to check out are:
```

File: cookbook/src/intro.md (hunk 5 above)
Comment (tcharding) [reply cookbook/src/intro.md:17]: thanks, covered above.

**Comment by tcharding** [comment]
Migrated to https://github.com/rust-bitcoin/rust-bitcoin.github.io/pull/4

**Pull request closed without merging**